core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,552 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Graag een aantal testcases toevoegen voor deze xslt (en eventueel kijken wat het iaf-test project voor plaatjes maakt), zeker de edge cases weet ik niet zeker of dit allemaal goed gaat